src/components/views/TitleFormInfo.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+import PropTypes from 'prop-types';
+
+import {
+  Checkbox,
+  Col,
+} from '@folio/stripes/components';
+
+const TitleFormInfo = ({
+  isSuppressFromDiscoveryEnabled,
+  name,
+}) => (
+  <div data-test-platform-form-info>
+    <div data-test-title-instance-name>
+      {name}
```

Comment:
Do we have to display the name here?

Reply:
no, that was probably only a dumb copy-and-paste from another formInfo component